crates/webidl/tests/expected/Event.rs

```diff
@@ -53,6 +53,124 @@ impl From<Event> for ::wasm_bindgen::JsValue {
         obj.obj
     }
 }
+#[no_mangle]
+#[allow(non_snake_case)]
+pub extern "C" fn __wbindgen_describe___wbg_f_stopPropagation_stopPropagation_Event() {
+    use wasm_bindgen::describe::*;
+    inform(FUNCTION);
+    inform(1u32);
+    <&Event as WasmDescribe>::describe();
+    inform(0);
+}
+impl Event {
+    #[allow(bad_style)]
+    #[cfg(all(target_arch = "wasm32", not(target_os = "emscripten")))]
+    pub extern "C" fn stopPropagation(&self) {
```

Comment:
Hm this looks a little fishy (the `extern "C"`), although it may be a preexisting bug, I'm not entirely sure